docs/docs-beta/docs/getting-started/installation.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: "Installing Dagster"
-description: "Learn how to install Dagster"
+title: 'Installing Dagster'
```

Comment:
we can probably just get rid of all quotes

Reply:
@yuhan here's a fast-follow to remove quotes for integration front-matter.
https://github.com/dagster-io/dagster/pull/24404